### DISCUSSION TITLE
Welcome to OpenLMS Discussions!

### DISCUSSION THREAD

**Discussion by Bhanu-code** [Bhanu-code/OpenLMS#15] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Bhanu-code** [comment]
we are open for ideas! please pitch!

**Comment by Reboot2004** [comment]
I would like to create figma design for login, registration as well as implement them

**Comment by Bhanu-code** [comment]
Hey, everyone thank you to show interest in contributing OpenLMS project. The idea behind OpenLMS is to provide an open source codebase for educational institutes who wants to create and manage their LMS, but want a basic boilder plate SDK to development. Institutions can use the OpenLMS SDK for their personal LMS providing scalability, smoothness, and secuirity managed. Institutional educators can teach, provide materials, conduct tests & assignments etc. Students can see their courses and subjects, complete assignments, give exams & tests, etc.

We need a figma design for the platform with pages like login, dahsboard, enrolled courses, progress, online tests, study materials etc.
